packages/wonder-blocks-button/src/components/button.tsx

```diff
@@ -264,7 +230,7 @@ export default class Button extends React.Component<Props> {
             router,
         );
 
-        const renderProp = (state: ClickableState, {...restChildProps}) => {
```

Comment:
The spread copies the object.  I don't think we need to do that here, but if people have concerns I can revert this change.